ably.d.ts

```diff
@@ -4399,6 +4399,7 @@ declare namespace Types {
      *
      * @param name - Event name.
      * @param data - Message payload.
+     * @param options - This parameter does not have any effect.
```

Comment:
I think we should go ahead and change it based on:
- It's only a breaking change to TS definitions so it's not gonna affect CDN users and break someone;'s app in production - they'll need to pull down the new version and compile again to see an error.
- If anyone's affected by this (and I'd be surprised if anyone was) then it's probably good to make them aware of it since they're probably mistaken about what it's actually doing.

Reply:
OK, cool, added [this commit](https://github.com/ably/ably-js/commit/f902f87fffb80caaf87180d094a1b47db37f053f) to address it.